resources/js/mosaic.tile.js

```diff
@@ -946,6 +944,9 @@ class Tile {
         }
 
         if (tile_config.tile_type === "field") {
+            window.__mosaic_saving_tile = true;
+            utils.loading.show();
```

Comment:
These both lines should IMO go directly before the `tiny.setContent` line - you have some `return` statements following which would let the app in a "save tile" state which maybe wouldn't be resolved.

But is `tiny.setContent` really worth to block something?‌ Shouldn't that be an immediate action?

Reply:
I've refactored the state to a pattern instance attribute instead of the global window state.
Also the possible stale state is fixed.
And I've removed the state from the immediate `setContent` action for `field` tiles... this works like it should.